reactioncommerce/reaction
# Fix on ProductAdmin meta textfields throwing errors on state change

**Resolves #2581** 

The error thrown was been caused by `this.state` not having a `products` field. However the reason why it was still been saved was because saving happens `onBlur`. So this actually saves regardless of the console error been thrown. All i had to do was re-arrange things a bit and take away the `update()` method. @jshimko the `update()` method actually still works but not sure if its still in use with the latest react because its been used as a standalone as `react-addons-update`.

**Test by**
- Running reaction from the  marketplace branch
- Log in as admin and go to product detail page
- Change to edit mode
- Click an edit pencil to open product admin panel
- Expand "Details" tab and start typing in any meta text field
- Observe that there's no console errors
